settings.gradle

```diff
@@ -1,3 +1,10 @@
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
```

Comment:
Do we use any plugins from `gradlePluginPortal()`? If we do, can we put it last (as recommended in the [Android docs](https://developer.android.com/build/optimize-your-build#gradle_plugin_portal))? Otherwise we can remove it completely.

Reply:
Yes, we use the `ktfmt-gradle` to use `googleStyle` for identation. Moved to the last position in the plugins list, as recommended.